ci/build_windows.py

```diff
@@ -106,29 +129,37 @@ class BuildFlavour(Enum):
         '-DCMAKE_BUILD_TYPE=Release')
 
     , 'WIN_GPU': (
+        '-DCMAKE_C_COMPILER=cl '
+        '-DCMAKE_CXX_COMPILER=cl '
         '-DUSE_CUDA=ON '
         '-DUSE_CUDNN=ON '
         '-DENABLE_CUDA_RTC=ON '
         '-DUSE_OPENCV=ON  '
+        '-DOpenCV_RUNTIME=vc15 '
+        '-DOpenCV_ARCH=x64 '
         '-DUSE_OPENMP=ON '
         '-DUSE_BLAS=open '
         '-DUSE_LAPACK=ON '
         '-DUSE_DIST_KVSTORE=OFF '
-        '-DMXNET_CUDA_ARCH="5.2" '
+        '-DMXNET_CUDA_ARCH="7.5" '
```

Comment:
This change warrants further discussion. Please change it back.

Reply:
This change sets the tested cuda architecture. The CI server now uses T4 GPUs which use 7.5 arch. Thus no, I can't change this back.

Reply:
Well where did we agree that we are dropping g3 instances? How is such a change in the scope of this PR?

Reply:
Let's keep using g3 instances for now then.